crypto/src/main/java/net/corda/v5/crypto/KeyUtils.java

```diff
@@ -0,0 +1,63 @@
+package net.corda.v5.crypto;
+
+import java.security.PublicKey;
+import java.util.Collections;
+import java.util.Set;
+
+public class KeyUtils {
+
+    private KeyUtils() {
+        // this class is never constructed; it exists for the static methods and data only
+    }
+
+    /**
+     * Constant specifying the maximum number of key lookup by id items.
+     */
+    public static int KEY_LOOKUP_INPUT_ITEMS_LIMIT = 20;
```

Comment:
It's used for instance in the `net.corda.data.crypto.wire.ops.flow.queries.ByIdsFlowQuery` Avro messsage. The number 20 is in the Avro docstring, and it is used in both the implementations and test in `corda-runtime-os`. However, it is reasonable to have access to a numeric constant value here, rather than make me put the constant in to their code.

Reply:
I think this should be internal. Nothing for the user to know how we batch the keys onto Kafka.

Reply:
The Avro messages are not internal; the number 20 is already in `corda-api`. So I regard it as public information. The client-side (with respect to Kafka) stuff we offer is, as far as  I know, best regarded as sample code.

Reply:
(The constant was originally defined in CryptoUitls)

Reply:
I agree with @kyriathar on this. I don't think this belongs in the API bytecode.

AVRO changes we can change over time, but API bytecode we can't. 
If we choose to change this to 25 as part of 5.1, why would that impact somebody's CorDapp ?

Reply:
Again, seems a shame to delay this PR, but we can do this now so let's do that.

Reply:
removed locally

Reply:
pushed